fuzz_plugin.py

```diff
@@ -92,8 +99,45 @@ def fuzz(install_dependencies: bool):
         reinitialize_containers()
 
         try:
-            if install_dependencies:
-                for dependency in dependencies:
+            if plugin_info_dict['from_file']:
```

Comment:
The logic to install either from SVN or from Wordpress plugin repository is already in line 134, let's keep various "ways of installation" close together

Reply:
Good idea. I put it that way because of the wrong assumption that zip plugins will not have dependencies. It also makes sense to move all subprocess to fuzzer_container.py as this seems to be the right place for all those generic&specific commands. Let me know what you think.

Reply:
Makes sense! Do you want to do this?